server/jetstream_api.go

```diff
@@ -555,10 +556,16 @@ func (s *Server) setJetStreamExportSubs() error {
 }
 
 func (s *Server) sendAPIResponse(ci *ClientInfo, acc *Account, subject, reply, request, response string) {
+	acc.trackAPI()
 	s.sendInternalAccountMsg(nil, reply, response)
 	s.sendJetStreamAPIAuditAdvisory(ci, acc, subject, request, response)
 }
 
+func (s *Server) sendAPIErrResponse(ci *ClientInfo, acc *Account, subject, reply, request, response string) {
+	acc.trackAPIErr()
+	s.sendAPIResponse(ci, acc, subject, reply, request, response)
```

Comment:
So an error is calling sendAPIResponse() which calls trackAPI(), which means that this will send 2 updates ok++ and err++?

Reply:
Thanks, I had it as total and err and changed it but did not update this. Will fix.